tools/vlsvdiff.cpp

```diff
@@ -277,6 +280,52 @@ bool HandleFsGrid(const string& inputFileName,
 
 }
 
+bool getFsgridDecomposition(vlsvinterface::Reader& file, std::array<int,3>& decomposition){
```

Comment:
Is it that you're keeping this also in this file because there's no parameters input for the override of decomposition?

Reply:
It is there because the ioread interfaces ParallelReader, which is not trivially compatible with vlsvinterface::Reader, and besides changing that introduces a whole lot of new dependencies for vlsvdiff.